app/(pages)/admin/submit-request/page.tsx

```diff
@@ -19,8 +19,22 @@ const SubmitRequestPage: React.FC = () => {
   const adminStore = useAdminStore();
   const router = useRouter();
   const activeAccount = useActiveAccount();
-  const [accessToken, setAccessToken] = useLocalStorage(LOCALSTORAGE_KEYS.ACCESS_TOKEN, null);
-  const [refreshToken, setRefreshToken] = useLocalStorage(LOCALSTORAGE_KEYS.REFRESH_TOKEN, null);
+  const [accessToken, setAccessToken] = useLocalStorage(
+    LOCALSTORAGE_KEYS.ACCESS_TOKEN,
+    ""
+  );
+  const [accessTokenExpiry, setAccessTokenExpiry] = useLocalStorage(
```

Comment:
we can store the accesstoken object instead of separately storing the token and the expiry.